airflow/utils/db.py

```diff
@@ -15,9 +15,12 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import importlib
 import logging
 import os
+import time
 
+from alembic.script import ScriptDirectory
```

Comment:
Please move this in to the wait function -- we don't import this module unless we have to.